<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageTableWidget</class>
 <widget class="QWidget" name="PageTableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1206</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PageTableWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QTableWidget" name="tbw_table">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">
/* 如下代码设置列表左上角那个格子的边框线 */
QTableCornerButton::section{ 
  border-top:0px solid #E5E5E5;
  border-left:0px solid #E5E5E5;
  border-right:0.5px solid #E5E5E5;
  border-bottom: 0.5px solid #E5E5E5;
  background-color:white;
}
/* 如下代码设置横向表格头的间隔线，有四个方向的间隔线,不需要间隔线的可以设置为0px */
QHeaderView::section{
  border-top:0px solid #E5E5E5;
  border-left:0px solid #E5E5E5;
  border-right:0.5px solid #E5E5E5;
  border-bottom: 0.5px solid #E5E5E5;
  background-color:white;
  padding:4px;
}
/* 如下代码设置纵向表格头的间隔线，有四个方向的间隔线, 不需要间隔线的可以设置为0px */
QHeaderView::section{
  border-top:0px solid #E5E5E5;
  border-left:0px solid #E5E5E5;
  border-right:0.5px solid #E5E5E5;
  border-bottom: 0.5px solid #E5E5E5;
  background-color:white;
  padding:4px;
}
/* 选中项目 */
QTableWidget::item:selected {
  background-color: rgb(225, 227, 230);  /*rgb(0, 159, 170) (225, 227, 230)*/
  border-radius: 3px; /*圆角度*/
}</string>
     </property>
     <property name="rowCount">
      <number>10</number>
     </property>
     <property name="columnCount">
      <number>6</number>
     </property>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="ComboBox" name="cbb_cnt">
       <property name="items_" stdset="0">
        <string>100条/页
1000条/页
10000条/页
100000条/页</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PushButton" name="btn_first">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>首页</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PushButton" name="btn_pre">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>上一页</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LineEdit" name="edt_cur_page">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="StrongBodyLabel" name="label">
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="BodyLabel" name="lab_cnt_page">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PushButton" name="btn_next">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>下一页</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PushButton" name="btn_last">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>尾页</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PushButton" name="btn_jump">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>跳转</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="PushButton" name="btn_export">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>导出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>StrongBodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
